src/command_modules/azure-cli-acs/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+2.0.26
+++++++
+* `az aks create` defaults to Standard_DS1_v2 sizes for VM nodes
```

Comment:
Recommend rewording this 

* `aks create` VM node size default changed from "Standard_D1_v2" to "Standard_DS1_v2".